vividus-plugin-rest-api/src/main/java/org/vividus/http/validation/model/ResourceValidation.java

```diff
@@ -18,17 +18,19 @@
 
 import java.net.URI;
 
+import org.apache.commons.lang3.tuple.Pair;
+
 public class ResourceValidation extends AbstractResourceValidation<ResourceValidation>
 {
-    public ResourceValidation(URI resourceUri)
+    public ResourceValidation(Pair<URI, String> uriOrError)
     {
-        super(resourceUri);
```

Comment:
```java
public ResourceValidation(URI resourceUri)
{
..
}

public ResourceValidation(String error)
{
..
}
```

Reply:
Fixed, but ResourceValidation(Pair<URI, String> uriOrError) should be still present for copying